src/Constants.js

```diff
@@ -26,6 +26,8 @@ const thirdPartyToHostEvents = {
   RESIZE_WIDGET: "RESIZE_WIDGET",
   USER_AVAILABLE: "USER_AVAILABLE",
   USER_UNAVAILABLE: "USER_UNAVAILABLE",
+  READY: "READY",
```

Comment:
We have onReady handler [here](https://private.hubteam.com/opengrok/xref/all/git.hubteam.com-HubSpot-calling-extensions-fe%23master/calling-integration-sdk-support/static/js/viewport/hooks/useStandardThirdPartyWidgetIframe.tsx#190) but where are we sending the READY event to HubSpot?